source_code/caches/caches.core

```diff
@@ -11,6 +11,8 @@ filesets:
             - separate_caches.sv
             - direct_mapped_tpf/direct_mapped_tpf_cache.sv
             - pass_through/pass_through_cache.sv
+            - l1/l1_cache.sv
+            - sram/sram.sv
         file_type: systemVerilogSource
 
```

Comment:
Add target for your TB simulation instead of shell scripts to run

Reply:
removed